### DISCUSSION TITLE
Does thunder client send the request through the http_proxy configurations of VS Code?

### DISCUSSION THREAD

**Discussion by mrdulin** [thunderclient/thunder-client-support#1208] [Q&A]
I have started a local server with `http://localhost:3000`, I set the `http_proxy` in VS Code. When I use Thunder client to send an HTTP POST request, got the error: 502 bad gateway.  I guess it is caused by the `http_proxy` configuration of VS Code. 

I found that the Thunder client has a "Exclude Proxy Host List" configuration, so I set the `localhost` hostname for it. The error is gone.

So the does thunder client send the request through the HTTP proxy first? The request workflow is like this:

Thunder  Client => HTTP Proxy => Target server(`http://localhost:3000` in my case)

**Comment by rangav** [comment]
Hi @mrdulin looks like those proxy not working correctly 

see this vscode docs to set correctly 
https://code.visualstudio.com/docs/setup/network
